### PR TITLE
Enable read-only access to policies for management and monitoring users

### LINKED ISSUE #156: View policies for users tagged under 'management' and 'monitoring' roles
As of March 2016, policies are visible (and also create, update and delete) only for those users who are tagged under the 'policymaker' and 'administrator' roles. Please consider the 'management' and 'monitoring' roles to allow viewing policies but not updating/deleting them.

### PR DESCRIPTION
**Pull request by hairyhum**
Fixes #156

### CODE REVIEW

File: priv/www/js/tmpl/policies.ejs [hunk 1]
```diff
@@ -27,7 +27,11 @@
 <% if (vhosts_interesting) { %>
      <td><%= fmt_string(policy.vhost) %></td>
 <% } %>
+<% if (user_policymaker) { %>
```
Comment (michaelklishin) [inline priv/www/js/tmpl/policies.ejs]: Please add the missing `is`. I do realise that `vhosts_interesting` and probably a few other places have the same issue but we shouldn't make it worse.

File: test/src/rabbit_mgmt_test_http.erl [hunk 2]
```diff
@@ -1650,7 +1650,7 @@ policy_permissions_test() ->
 
     http_put("/users/admin",  [{password, <<"admin">>},
                                {tags, <<"administrator">>}], [?CREATED, ?NO_CONTENT]),
-    http_put("/users/mon",    [{password, <<"monitor">>},
+    http_put("/users/mon",    [{password, <<"mon">>},
```
Comment (michaelklishin) [inline test/src/rabbit_mgmt_test_http.erl]: Does this value need to change?

**Comment by michaelklishin** [comment]
If we expose policies, we might as well expose parameters.

File: test/src/rabbit_mgmt_test_http.erl (hunk 2 above)
Comment (hairyhum) [inline test/src/rabbit_mgmt_test_http.erl]: Yes, because in `http_get` password should be same as username.

**Comment by hairyhum** [comment]
Should we show parameter based pages like federation upstreams and shovel management if we expose parameters?

**Comment by michaelklishin** [comment]
Those come from separate plugins. I'd say we should but please file separate issues for federation-management and shovel-management.

**Comment by uvzubovs** [comment]
Please be careful and not expose upstreams because URIs contain federation user password.

**Comment by uvzubovs** [comment]
Would you please help me understand the status of this item? Was it completed or deferred? I am trying to get policy info in 3.6.5, and it seems that I still need policymaker tag. Thanks!

**Comment by dcorbacho** [comment]
Users with management/monitoring tags are allowed to list policies and their details using `/api/policies`, but `/api/policies/vhost/name` is restricted to policymakers as that URI allows to modify (create/delete) the policy.

**Pull request merged**